examples/thread-test/content.js

```diff
@@ -79,18 +79,33 @@ InboxSDK.load(2, 'thread-example').then((sdk) => {
 
       threadView.addSubjectButton({
         tooltip: 'Hi a thing',
-        iconUrl: `//ssl.gstatic.com/ui/v1/icons/mail/rfr/ic_open_in_full_16px_1x.png`,
+        iconUrl: `//ssl.gstatic.com/ui/v1/icons/mail/gm3/1x/forward_baseline_nv700_20dp.png`,
```

Comment:
100% a nit, but: for our devs on higher res screens. Using 2x instead of 1x is a higher pixel density image. It doesn't work for all gmail icons. I think they're moving towards variable fonts for icons, slowly.

```suggestion
        iconUrl: `//ssl.gstatic.com/ui/v1/icons/mail/gm3/2x/forward_baseline_nv700_20dp.png`,
```